src/tests/ftest/util/performance_test_base.py

```diff
@@ -17,6 +17,7 @@
 from exception_utils import CommandFailure
 
 
+# TODO dmg system query as non-json to reduce log clutter # pylint: disable=fixme
```

Comment:
This comment has since been removed entirely in master
```suggestion
```

Reply:
Oh yeah, bad merge.  I'll fix.